gatt_blacklist.txt

```diff
@@ -23,6 +23,11 @@
 # Block access to standardized unique identifiers, for privacy reasons.
 00002a25-0000-1000-8000-00805f9b34fb
 
+# Random UUID for testing a characteristic excluding reads.
+# Should be removed from this blacklist if a characteristic excluding reads
+# is added.
+bad1c9a2-9a5b-4015-8b60-1579bbbf2135 exclude-reads
```

Comment:
We can keep it in the implementation, though it will be in the layout tests and test devices spec -- implying that all implementations will have these hard coded into their blacklists if they use the portable tests.